doc/ref_model/chapters/glossary.md

```diff
@@ -73,7 +74,23 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
 
 <a name="11.1.4"></a>
-### 1.1.4 Other Referenced Terminology
+### 11.1.4 Container Related Terminology
+
+>_*Note:*_ Relevant terms are added here from RA2. Most of these term definitions are taken from [Kubernetes glossary](https://kubernetes.io/docs/reference/glossary) but in some cases should be made independent from Kubernetes as a specific container orchestration engine.
+
+- **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
+- **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
+- **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
+- **Kubernetes Cluster**: A set of machines, called nodes, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master node.
```

Comment:
In kubernetes, a node is a "worker machine" -- see term "Kubernetes Node" below Please delete "node" from "master node" and also from similar use in "Kubernetes Master."